osu.Server.Spectator/Hubs/ServerMultiplayerRoom.cs

```diff
@@ -27,13 +29,22 @@ public MatchTypeImplementation MatchTypeImplementation
             }
         }
 
-        public ServerMultiplayerRoom(long roomId, IMultiplayerServerMatchCallbacks hubCallbacks)
+        public readonly MultiplayerQueue QueueImplementation;
```

Comment:
Shall we drop the `Implementation` suffix on this (and the existing usage)? Doesn't seem like it's required.